### PR TITLE
Redux implementation

### PR DESCRIPTION
**Pull request by serengia**
Hello. Kindly review my request to merge `redux-implementation` branch to `development` branch.
The branch contains the following features:
- Implementation of redux toolkit store
- Implementation of books slice
- Implementation of categories slice
- Provided store to the root of the application

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+import booksSlice from "./books/booksSlice";
+
+const store = configureStore({
+  reducer: {
+    books: booksSlice,
+    categories: "",
```
Comment (TedLivist) [inline src/redux/store.js]: - [x] Kindly import the categories reducer as well and set up its reducer here also.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @serengia,

Good job so far!
✔️ Good documentation
✔️ Implementation of slices and reducers for books
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Thank you for your feedback. I will do the changes as soon as possible.

**Review by gemmen29 — APPROVED** [review]
Hi @serengia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by serengia** [comment]
Thank you.

**Pull request merged**